packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPageConnector.spec.js

```diff
@@ -5,36 +5,105 @@ import sinon from 'sinon'
 import ProjectAboutPageConnector from './ProjectAboutPageConnector'
 import ProjectAboutPage from './ProjectAboutPage'
 
-describe('Component > ProjectAboutPageConnector', function () {
+describe('Component > ProjectAboutPageConnector', () => {
```

Comment:
Non-blocking comment, and just a FYI in case you weren't aware: [arrow functions are discouraged](https://mochajs.org/#arrow-functions), but depends on if you need mocha's context. We tend to default using `function() {}` for this reason just for safety.

Reply:
I did not know that, but I'll make sure to avoid them in the future!